code/flusion/retrospective-experiments/run-flusion-retrospective.R

```diff
@@ -0,0 +1,26 @@
+# Create flusion ensembles for all combinations of
+# reference date and subsets of component models selected from
+# UMass-gbq_qr, UMass-gbq_qr_no_level, and UMass-sarix
+# Run with code/flusion/retrospective-experiments as you working directory
```

Comment:
I was slightly confused here, because I wasn't sure if "Run with" meant to run something in an R session or from the command line (I figured it out using the c[omment in the other file](https://github.com/reichlab/flusion/pull/33/files#diff-e55908b5f1500818d252389a8d23a1e2bdcf5297bcdcfe63859f5cf0f8638dacR15) as guidance)